tests/integration/deprecations/test-support-test.js

```diff
@@ -0,0 +1,16 @@
+import { test, module } from 'qunit';
+import require from 'require';
+
+module('Deprecation | test-support');
+
+test('import from test-support leads to the deprecation', function(assert) {
+  // Currently addon-exports-test statically imports `dummy/tests/page-object`.
+  // That leads to pre-loading and caching of the `dummy/tests/page-object` module
+  // on the tests startup phase.Then if we tryto  access this module again
+  // w/o cleaning the cache we won't be able to catch a deprecation.
+  window.require.unsee('dummy/tests/page-object');
+
+  require('dummy/tests/page-object');
+
+  assert.expectDeprecation(`Importing from "test-support" is now deprecated. Just import directly from the "ember-cli-page-object" module instead.`);
```

Comment:
Could this assertion somehow live in `addon-exports-test`, so that once we remove support for importing from `test-support`, we only have to update that one test?

Reply:
My intention was to remove the whole `tests/integration/deprecations` directory once we switch to 2.0. Actually I'd prefer to move `test-support` import tests from `addon-exports-tests` here but these tests are not easy to extract..

Reply:
Yeah, I like your idea of moving the `test-support` import tests here...that addresses my question about reducing the # of files that will need to be updated later. It's not really that big a deal, though, so I think we're fine either way.